docs/internal-documentation/maintenance/IssueLabels.md

```diff
@@ -17,38 +18,49 @@ when applicable.
 One label from *Priority* could be attached to an issue, or else the issue is considered
 to have normal priority. A priority label could also be set on a pull request, if applicable.
```

Comment:
A priority label could also be set on a pull request, if applicable. -> Similarly, the same could be done for a pull request.

Reply:
Updated.